mtgjson5/resources/sealed_name_fixes.json

```diff
@@ -1,15 +1,55 @@
 {
+    "All 5": "Set of 5",
+    "Set of Five": "Set of 5",
+    " Box Set": "",
+    "Collector Pack": "Collector Booster Pack",
     "DotP": "Duels of the Planeswalkers",
     "M15 Prerelease": "Magic 2015 Prerelease",
-    "All 5": "Set of 5",
     "M10 ": "",
     "M11 ": "",
     "M12 ": "",
     "M13 ": "",
     "M14 ": "",
     "M15 ": "",
+    "Core Game": "Two Player Starter",
     "2 Player": "Two Player",
+    "2-Player": "Two Player",
     "Unlimited Edition": "Unlimited",
-    "The Brothers' War": "The Brothers War",
-    "Set of Two": "Set of 2"
+    "3rd Edition Revised": "Revised Edition",
+    "4th Edition": "Fourth Edition",
+    "5th Edition": "Fifth Edition",
+    "6th Edition": "Classic Sixth Edition",
+    "7th Edition": "Seventh Edition",
+    "8th Edition": "Eighth Edition",
+    "9th Edition": "Ninth Edition",
+    "10th Edition": "Tenth Edition",
+    "Magic 2010": "2010 Core Set",
+    "Magic 2011": "2011 Core Set",
+    "Magic 2012": "2012 Core Set",
+    "Magic 2013": "2013 Core Set",
+    "Magic 2014": "2014 Core Set",
+    "Magic 2015": "2015 Core Set",
+    "The Brothers War": "The Brothers' War",
+    "Shards of Alara Block": "Shards of Alara",
+    "Set of Two": "Set of 2",
+    "[Set of 2]": "Set of 2",
+    "Decks - Set": "\u2691",
+    "Decks Set": "\u2691",
```

Comment:
this is a bit confusing IMO, I'd just repeat "Decks - Set" for each

Reply:
What this block is actually cleaning up is "deck" vs "decks". Card Kingdom likes to use `{set} commander decks {name}` where TCGplayer uses `{set} commander deck {name}`.

At the same time, I can't just replace "decks" with "deck" globally because then `commander decks set of 2` type products look like a mess. 

This bit of code finds anywhere that is a set of decks, turns that phrase into a searchable flag character, then deals with anywhere that Deck is mutable, then turns the flag character back into `decks - set`.

Reply:
hmm yeah it's a mess xD
could it at least be documented somewhere?

Reply:
That's the plan eventually. I wish JSON allowed for comments.

Reply:
what i do for json comments is makeup my own entry on top of the necessary one, so like

```json
    "[Set of 2]": "Set of 2",
    "COMMENT1": "this is because xxxx",
    "Decks - Set": "\u2691",
    "Decks Set": "\u2691",
```

not pretty but gets the job done